sdk/include/platform/sunburst/platform-spi.hh

```diff
@@ -0,0 +1,92 @@
+#pragma once
+#include <cdefs.h>
+#include <stdint.h>
+
+/**
+ * A Simple Driver for the Sonata's GPIO.
+ *
+ * Documentation source can be found at:
+ * https://github.com/lowRISC/sonata-system/blob/1a59633d2515d4fe186a07d53e49ff95c18d9bbf/doc/ip/spi.md
+ *
+ * Rendered documentation is served from:
+ * https://lowrisc.org/sonata-system/doc/ip/spi.html
+ */
+struct SonataSpi
+{
+	// Interrupt State Register
+	uint32_t intrState;
```

Comment:
Please use full words for variable and function names. Our coding guidelines to avoid abbreviations. You know what intr means but the next person looking at this file or seeing it pop up in autocomplete may not.

Same for most of the other fields and methods here.

Reply:
I had followed the naming of [the hardware documentation](https://lowrisc.org/sonata-system/doc/ip/spi.html), but I agree renaming to be more explanatory would probably be better.